assets/app/scripts/directives/logViewer.js

```diff
@@ -71,8 +71,8 @@ angular.module('openshiftConsole')
 
 
 
-
-            var updateScrollLinks = function() {
+            // is just toggling show/hide, nothing else.
+            var toggleScrollLinks = function() {
```

Comment:
Agree on set... in this directive I definitely prefer descriptive though, so that may be good.